spec/rspec/filtering_spec.rb

```diff
@@ -16,6 +16,16 @@
       ).as_null_object
     end
 
+    context 'when filter manager has no inclusions defined' do
+      let(:files) { %w[./spec/foo_spec.rb] }
+      let(:inclusions) { {} }
+
+      it 'should not change any filtering rules' do
+        remove_unnecessary_filters
+        expect(inclusions[:ids]).to be_nil
```

Comment:
This is testing that the hash created in line 21 is the hash created in line 21. Probably better to do something like
```ruby
expect { remove_unecessary_filters }.not_to raise_error
```

Reply:
it tests that the hash wasn't changed.